hack/release-tool.py

```diff
@@ -468,6 +469,130 @@ def reimport(ctx, namespace, imagestream, execute):
+                        target_component_pullspec = tag['from'].name
+                        break
+
+                if not target_component_pullspec:
+                    logger.error(f'Unable to find component image {component_name} in {nightly_imagestream.qname()}')
+                    return
+
+                logger.info(f'Reverting tag {nightly_components.major_minor} nightly component {component_name} to reference nightly {to_nightly} image: {target_component_pullspec}')
+
+                art_imagestream = oc.selector(f'imagestream/{nightly_components.art_imagestream_name}').object(ignore_not_found=False)
+                cmd_args = ['--import-mode=PreserveOriginal', '--source=docker', target_component_pullspec, f'{art_imagestream.name()}:{component_name}']
+
+                if execute:
+                    oc.invoke('tag', cmd_args=cmd_args)
+                    logger.info('The tag has been reverted. Note that the next ART update will restore the tag. Use "lock", before reverting, to prevent this.')
+                else:
+                    logger.info(f'[dry-run] oc tag with {cmd_args}')
+
+        except (ValueError, OpenShiftPythonException, Exception):
+            logger.error(f'Unable to to process perform revert: of {nightly_components.major_minor} nightly component {component_name}')
+            raise
+
+
+def bypass(ctx, nightly: str, trigger: str, execute):
+    nightly_components = parse_nightly_components(nightly_name=nightly)
+    if not nightly_components:
+        logger.error(f'{nightly} does not match expected nightly naming convention')
+        return
+
+    with oc.options(ctx), oc.tracking(), oc.timeout(5 * 60):
+        try:
+            with oc.project(nightly_components.art_imagestream_namespace):
+                nightly_istag = oc.selector(f'imagestreamtag/{nightly_components.release_imagestream_name}:{nightly}').object()
+                annotations = {
+                    'release.openshift.io/bypass': 'true'
+                }
+
+                if execute:
+                    nightly_istag.annotate(annotations=annotations, overwrite=True)
+                    logger.info(f'Annotated {nightly_istag.qname()} as bypass')
+                else:
+                    logger.info(f'[dry-run] oc annotate {nightly_istag.qname()} with annotations {annotations}')
+
+                if trigger:
+                    art_imagestream = oc.selector(f'imagestream/{nightly_components.art_imagestream_name}').object(ignore_not_found=False)
+                    tag_cmd_args = ['--import-mode=PreserveOriginal', '--source=docker', 'registry.access.redhat.com/ubi9', f'{art_imagestream.name()}:trigger-release-controller']
+                    untag_cmd_args = ['-d', f'{art_imagestream.name()}:trigger-release-controller']
+
+                    if execute:
+                        oc.invoke('tag', cmd_args=tag_cmd_args)
+                        oc.invoke('tag', cmd_args=untag_cmd_args)
```

Comment:
Should we sleep for a little bit in between these calls?

Reply:
Added